SasquatchMac/SasquatchMac/Constants.h

```diff
@@ -4,6 +4,17 @@ static NSString *const kMSMainStoryboardName = @"SasquatchMac";
 static NSString *const kMSUserIdKey = @"userId";
 static NSString *const kMSLogTag = @"[SasquatchMac]";
 
+static NSString *const kSASCustomizedUpdateAlertKey = @"kSASCustomizedUpdateAlertKey";
+static NSString *const kMSChildTransmissionTargetTokenKey = @"kMSChildTransmissionTargetToken";
+static NSString *const kMSTargetToken1 = @"602c2d529a824339bef93a7b9a035e6a-"
```

Comment:
This fomatting can't be right, I can see below in lines 19-20 that the split strings are aligned after the equals sign.  All these strings are also being split in different places and at different column numbers.  Try running the formatter again or match the formatting already in the file.  (Personal preference is that all these strings are 1-line, but try to go with the formatter)

Reply:
Fixed.